recipe/meta.yaml

```diff
@@ -1,23 +1,25 @@
-{% set version = "2.3.1" %}
+# https://pagure.io/python-daemon/issue/94
+{% set version = "3.0.1" %}
 
 package:
   name: python-daemon
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/p/python-daemon/python-daemon-{{ version }}.tar.gz
-  sha256: 15c2c5e2cef563e0a5f98d542b77ba59337380b472975d2b2fd6b8c4d5cf46ca
+  sha256: 6c57452372f7eaff40934a1c03ad1826bf5e793558e87fef49131e6464b4dae5
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script: {{ PYTHON }} -m pip install . --no-deps -vv --no-build-isolation
   skip: True  # [win or (linux and s390x)]
 
 requirements:
```

Comment:
https://pagure.io/python-daemon/blob/main/f/setup.py#_67

```
setup_requires=[

  "docutils",
  "packaging",
  "setuptools",
  ],

 install_requires=[
   "setuptools >=62.4.0",
   "packaging",
   "lockfile >=0.10",
   ],
```

Reply:
This is interesting as well. https://pagure.io/python-daemon/issue/93

Reply:
definitely